cli/Sources/TuistGenerator/Generator/ProjectDescriptorGenerator.swift

```diff
@@ -286,11 +286,33 @@ final class ProjectDescriptorGenerator: ProjectDescriptorGenerating {
 
                 pbxproj.add(object: reference)
 
-                if let existingPackageGroup = try pbxproj.rootGroup()?.group(named: "Packages") {
-                    existingPackageGroup.children.append(reference)
+                guard let root = try pbxproj.rootGroup() else { continue }
```

Comment:
We need a test for this piece of business logic testing all the possible scenarios.